assets/scss/base.scss

```diff
@@ -62,7 +62,7 @@ body {
 h1, h2, h3, h4, h5, h6 {
   color: $gray9;
   letter-spacing: -0.02rem;
-  font-weight: 700;
+  font-weight: bold;
```

Comment:
This project only uses two font weights: 400 and 700 (per font declarations above).

Hence, it's cleaner (and less error-prone) to just use font weights "normal" and "bold" everywhere.